bot.py

```diff
@@ -1,545 +1,755 @@
+        query = message.text.replace('/wps', '').replace(f'@{BOT_USERNAME}', '').strip()
+        if query == '':
+            await message.reply(lng.getmess(message, 'empty'), parse_mode = "Markdown")
+        else:
+            try:
+                mac = [m for m in re.findall(MAC, query)][0]
+            finally:
+                gen = wpspin.WPSpin()
+                gen = gen.getAll(mac)
+                built = f'<code>{mac}</code>\n'
+                built += SEPARATOR
+                for pin in gen:
+                    built += f'<code>{pin["name"]}</code>\n{lng.getmess(message, "w_value")}: <code>{pin["pin"]}</code>\n'
+                    built += SEPARATOR
+                await message.reply(built, parse_mode = "HTML")
+
+"""
+@dp.message(commands = ['whatis'])
+async def who_mess(message: types.Message):
+    if users.getstatus(message) == Status.AUTH or users.getstatus(message) == Status.MAYDAY:
+        query = message.text.replace('/whatis ', '').replace(f'@{BOT_USERNAME}', '').strip()
+        if query == '':
+            await message.reply(lang.get(ulang(uid), 'empty'), parse_mode = "Markdown")
+        else:
+            try:
+                mac = [m.upper().replace('-', ':') for m in re.findall(MAC, query)][0]
+            finally:
+                s = requests.Session()
+                s.post(f'{SERVICE_URL}/user.php?a=login', 
+                    data = {'login': users.data[str(message.from_user.id)]['login'], 
+                        'password': users.data[str(message.from_user.id)]['pass']})
+                get = s.post(f'{SERVICE_URL}/3wifi.php?a=devicemac', data = {'bssid': mac}).json()
+                s.get(f'{SERVICE_URL}/user.php?a=logout6{s.get(f"{SERVICE_URL}/user.php?a=token").json()["token"]}')
+                if get['result'] == True:
+                    mess = f'`{mac}` {lng.getmess(message, "who")}:\n'
+                    mess += SEPARATOR
+                    maxi = 10
+                    limiter = 0
+                    for s in get['scores']:
+                        if limiter < maxi:
+                            limiter += 1
+                            mess += f'`{s["name"]}`\n{lng.getmess(message, "w_score")}: `{s["score"] * 100:.2f}%`\n'
+                            mess += SEPARATOR
+                    await message.reply(mess, parse_mode = "Markdown")
+                else:
+                    await message.reply(lng.getmess(message, 'nothing'), parse_mode = "Markdown")
+"""
+
+@dp.message(commands = ['where'])
+async def where_mess(message: types.Message):
+    if users.getstatus(message) == Status.AUTH or users.getstatus(message) == Status.MAYDAY:
+        query = message.text.replace('/where ', '').replace(f'@{BOT_USERNAME}', '').strip()
+        if query == '':
+            await message.reply(lang.get(ulang(uid), 'empty'), parse_mode = "Markdown")
+        else:
+            try:
+                mac = [m.upper().replace('-', ':') for m in re.findall(MAC, query)][0]
```

Comment:
```suggestion
                mac = [m for m in re.findall(MAC, query)][0]
```
Всё то же про Мак адреса